apps/src/code-studio/pd/workshop_dashboard/workshop_index.jsx

```diff
@@ -65,12 +65,13 @@ export default class WorkshopIndex extends React.Component {
 
   render() {
     const showOrganizer = this.permission.isWorkshopAdmin;
+    const canDelete = this.permission.isWorkshopAdmin || this.permission.isOrganizer;
 
     return (
       <div>
         <h1>Your Workshops</h1>
         <ButtonToolbar>
-          {(this.permission.isWorkshopAdmin || this.permission.isOrganizer) &&
+          {(this.permission.isWorkshopAdmin || this.permission.isOrganizer || this.permission.isCsfFacilitator) &&
```

Comment:
How about calling this out above as `const canCreate`, like we do for `canDelete`, for consistency and readability?